README.md

```diff
@@ -4,26 +4,36 @@ A modified version of the InfluxData 'TICK' Stack which uses Grafana as its UI a
 
 ## Requirements
 
+- Linux system with a Graphical User Interface
+
+- Firefox
+
 - Docker (v18.06+)
 
 - A functional Kubernetes cluster (v.1.13+)
 
-    > [Set up a Kubernetes Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network)
+    > [Set up a Kubernetes Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/)
 
 - 2GB RAM or more per node
 
 - C Compiler
 
+- Python
+
 - Linux "Make"
 
 > Recommended: Kubernetes v1.13 cluster running on three Ubuntu 18.04 Server VMs
 
 ## Installation
 
-Clone the repo and dive in!
+Clone the repository and dive in!
 
 `git clone https://github.com/Eliforbes42/cronus-monitoring.git`
 
+Installation will take place on the Master Node of your Kubernetes Cluster.
```

Comment:
Probably should direct them to the master node before telling them to clone the repo, I could see that potentially causing some confusion if they cloned it locally before reading down the guide further.